modules/extensions-xml-support/pom.xml

```diff
@@ -137,6 +139,13 @@
             <classifier>mule-plugin</classifier>
         </dependency>
         <dependency>
+            <groupId>org.mule.modules</groupId>
+            <artifactId>mule-module-jms</artifactId>
+            <version>${project.version}</version>
+            <scope>provided</scope>
```

Comment:
actually, they should be `<scope>test</scope>`. But the runner ask them to be `<scope>provided</scope>` or it fails